templates/CRM/Birthdays/Form/Settings.tpl

```diff
@@ -0,0 +1,196 @@
+{* HEADER *}
+
```

Comment:
Please wrap the entire template in `{crmScope extensionKey='de.systopia.birthdays'} ... {/crmScope}` for `{ts}` calls to pick up the scope.